src/app/app.component.html

```diff
@@ -10,10 +10,10 @@
             <li *ngIf="auth.authenticated()">
                 <a materialize="dropdown" class="dropdown-button" data-beloworigin="true" data-hover="true" href="#!" data-activates="dropdownproject">{{auth.getCurrentDisplayName()}}<i class="material-icons right">arrow_drop_down</i></a>
                 <ul id="dropdownproject" class="dropdown-content">
-                    <li><a routerLink="/account/{{auth.getCurrentUserId()}}" routerLinkActive="active" class="black-text text-lighten-2">My Account</a></li>
-                    <li  *ngIf="auth.isVolunteer()"><a routerLink="/profile/{{auth.getCurrentUserId()}}" routerLinkActive="active" class="black-text text-lighten-2">My Profile</a></li>
+                    <li><a routerLink="/account/{{currentUserId}}" routerLinkActive="active" class="black-text text-lighten-2">My Account</a></li>
```

Comment:
Like I said in the email I sent, I've tried this before and I get an exception about 
EXCEPTION: Error in ./AppComponent class AppComponent - inline template:22:27 caused by: Cannot read property '**outlets**' of null
ErrorHandler.handleError @ error_handler.js:54
next @ application_ref.js:348
schedulerFn @ async.js:93
SafeSubscriber.__tryOrUnsub @ Subscriber.js:234
SafeSubscriber.next @ Subscriber.js:183
Subscriber._next @ Subscriber.js:125
Subscriber.next @ Subscriber.js:89
Subject.next @ Subject.js:55
EventEmitter.emit @ async.js:79
NgZone.triggerError @ ng_zone.js:333
onHandleError @ ng_zone.js:294
ZoneDelegate.handleError @ zone.js:338
Zone.runTask @ zone.js:169
ZoneTask.invoke @ zone.js:420
data.args.(anonymous function) @ zone.js:1581

So in the meantime, I'll leave it this way until I get a chance to look into this further.